chainer/training/extensions/evaluator.py

```diff
@@ -193,3 +193,8 @@ def evaluate(self):
             summary.add(observation)
 
         return summary.compute_mean()
+
+    def finalize(self):
```

Comment:
Could you please copy the equivalent code (and pydoc) from Updater?
In the imagenet example code, `finalize` of `train_iter` is called from there.

https://github.com/chainer/chainer/blob/v4.0.0b2/chainer/training/updater.py#L169-L178

(Note: iterator base class `chainer.dataset.Iterator` defines `finalize` method, so `hasattr` is not needed)

Reply:
Thank you for the review. Pushed 74c144f .